packages/@o3r/eslint-plugin/src/rules/template/no-inner-html/no-inner-html.spec.ts

```diff
@@ -1,22 +1,26 @@
+import {
+  meta,
+  parseForESLint,
+} from '@angular-eslint/template-parser';
 import {
   RuleTester,
 } from '@angular-eslint/test-utils';
-import {
-  templateParser,
-} from 'angular-eslint';
 import noInnerHTMLRule, {
   name,
 } from './no-inner-html';
 
 const ruleTester = new RuleTester({
   languageOptions: {
-    parser: templateParser
+    parser: {
```

Comment:
It seems to be exactly the same, no ?
https://github.com/angular-eslint/angular-eslint/blob/main/packages/angular-eslint/src/index.ts#L13
![image](https://github.com/user-attachments/assets/07520add-21ad-4c14-886d-24afdd6adaea)

Reply:
Like `typescript-eslint`, `angular-eslint` built a package where you should have everything accessible instead of needed to know in which package is what